### PR TITLE
Update install & doc by using latest repo from ipex

### PR DESCRIPTION
**Pull request by xwu99**

### CODE REVIEW

File: dev/docker/Dockerfile.cpu_and_deepspeed [hunk 1]
```diff
@@ -29,8 +29,8 @@ COPY ./MANIFEST.in .
 
 RUN mkdir ./finetune && mkdir ./inference
 
-RUN --mount=type=cache,target=/root/.cache/pip pip install -e .[cpu,deepspeed] -f https://developer.intel.com/ipex-whl-stable-cpu \
-    -f https://download.pytorch.org/whl/torch_stable.html
+RUN --mount=type=cache,target=/root/.cache/pip pip install -e .[cpu,deepspeed] --extra-index-url https://download.pytorch.org/whl/cpu \
+    --extra-index-url https://pytorch-extension.intel.com/release-whl/stable/cpu/us/
```
Comment (carsonwang) [inline dev/docker/Dockerfile.cpu_and_deepspeed:33]: Why some use "--index-url  --extra-index-url", some use "--extra-index-url --extra-index-url"? Is there any difference? But let's still better to use the same format.

Comment (xwu99) [reply dev/docker/Dockerfile.cpu_and_deepspeed:33]: --index-url will replace the default https://pypi.python.org/simple, should use --extra-index-url in addition to default.

**Comment by xwu99** [comment]
@jiafuzha I did not change bigdl case, it failed anyway, looks sth wrong from upstream.

**Comment by jiafuzha** [comment]
> @jiafuzha I did not change bigdl case, it failed anyway, looks sth wrong from upstream.

I need to change model id to mpt-7b-chat according to https://github.com/intel-analytics/BigDL/issues/10177. Let me submit a PR.

**Comment by xwu99** [comment]
will rebase after gptj-6b & bigdl cases are fixed.

**Review by carsonwang — APPROVED** [review]
LGTM. Thanks for improving this!

**Pull request merged**